docs/user-guides/benchmark.rst

```diff
@@ -0,0 +1,294 @@
+CLIP Benchmark
+==============
+
+In order to evaluate the performance of different CLIP models, we conducted a benchmark on a series of tasks using different datasets. 
+We fork the `CLIP benchmark repo <https://github.com/LAION-AI/CLIP_benchmark>`_, slightly modify the codebase and apply it to all Vision Transformers (ViT) and ResNet (RN) CLIP models. 
+You can find the benchmark results in the following tables. 
+The best results are highlighted in bold. 
+They can be used as a guide to choose the best model for your application.
+
+Basic statistics
+----------------
+
+In the table below, we include the disk usage (in delta) and the peak RAM and VRAM usage (in delta) when running on a single Nvidia TITAN RTX GPU (24GB VRAM) for a series of text and image encoding tasks with ``batch_size=8`` using PyTorch runtime.
```

Comment:
please also specify the cpu/memory info, also the version of `jina`, `cas`, `docarray`...